templates/rpm.erb

```diff
@@ -43,7 +43,7 @@ Epoch: <%= epoch %>
 <% end -%>
 Release: <%= iteration or 1 %><%= "%{?dist}" if attributes[:rpm_dist] %>
 <%# use the first line of the description as the summary -%>
-Summary: <%= description.split("\n").first || "_" %>
```

Comment:
Yes. The behavior is still the same, just the location has changed. A summary method was added which defaults to the first line of the description, or otherwise the summary argument. If no summary or description are passed, then the underscore is used. See `lib/fpm/package/rpm.rb` in this PR for it.